packages/frontend/src/components/common/modal/ChartCreateModal.tsx

```diff
@@ -106,6 +111,12 @@ const ChartCreateModal: FC<ChartCreateModalProps> = ({
         showSpaceInput,
     ]);
 
+    const handleSaveChartInDashboard = () => {
+        sessionStorage.clear();
+        // TODO: actually save the chart in the dashboard before redirecting
+        history.push(`/projects/${projectUuid}/dashboards/${dashboardUuid}`);
```

Comment:
Let's implement this part, where we send the chart data to the dashboard page and save the dashboard tile with it.